bin/system-info

```diff
@@ -1,30 +1,236 @@
-#! /bin/bash
+#!/bin/bash
+export LC_ALL=C
 
-# All whitespace below is required for correct parsing in protondb
-echo "
-System Info:
+# Scaffold some hefty gaurd rails around ensuring DISPLAY works
+if [ -z "${DISPLAY}" ]; then
+    # Hunt for an accessible DISPLAY
+    for LOOP in {0..5}; do
+        if env DISPLAY=:"${LOOP}" xdotool sleep 0 >&/dev/null; then
+            export DISPLAY=:${LOOP}
+            break
+        fi
+        if [ "${LOOP}" -eq 5 ]; then
+            echo "ERROR! Failed to find an accessible display."
+            exit 1
+        fi
+    done
+else
+    # Ensure DISPLAY is accessible
+    if ! xdotool sleep 0 >&/dev/null; then
+        echo "ERROR! ${DISPLAY} is not accessible."
+        exit 1
+    fi
+fi
 
+# Dump data
+for CMD in glxinfo xdpyinfo xrandr; do
```

Comment:
We should make chimera dependent on `xdpyinfo` for this or at least an optional dependency. Maybe the script should check if it present?

Reply:
The script does check for the availability of `xdpyinfo` 🙂 First here:

https://github.com/flexiondotorg/chimera/blob/system-info/bin/system-info#L27

Then here:

https://github.com/flexiondotorg/chimera/blob/system-info/bin/system-info#L75

And here:

https://github.com/flexiondotorg/chimera/blob/system-info/bin/system-info#L101

I also submitted a pull request to update the PACKAGE manifest here:

https://github.com/ChimeraOS/chimeraos/pull/366

Reply:
Great. Thanks for that.
 The package dependency should be managed in packaging of chimera app. Not necessary to add to the image, would be added by pacman when needed.